string setting
string value
---
bool succes
string error